apps/server/src/modules/h5p-editor/contentStorage/contentStorage.spec.ts

```diff
@@ -174,22 +266,34 @@ describe('ContentStorage', () => {
 		});
 		describe('WHEN content can not be added error', () => {
 			it('should throw new error', async () => {
-				const { metadata, content, user } = setup();
-				jest.spyOn(fsPromiseMock, 'writeFile').mockImplementationOnce(() => {
+				const { metadata, content, user, contentId } = setup();
+				s3ClientAdapter.create.mockImplementationOnce(() => {
 					throw new Error('Could not write file');
 				});
+				s3ClientAdapter.get.mockRejectedValue(new NotFoundException('NoSuchKey'));
 				try {
-					await service.addContent(metadata, content, user);
+					await service.addContent(metadata, content, user, contentId);
+				} catch (err) {
+					expect(err).toBeInstanceOf(Error);
+				}
+			});
+		});
+		describe('WHEN content already exists', () => {
+			it('should throw new error', async () => {
+				const { metadata, content, user, contentId } = setup();
+				s3ClientAdapter.get.mockRejectedValue(new Error('FoundTooMany'));
```

Comment:
Same here as above. Please take care what you are mocking and test for errors properly! Also below there are some alike mistakes. Please correct that in the whole file!